api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -2751,6 +2751,14 @@ public void unvoidPatient_shouldUnvoidPerson() throws Exception {
 		Assert.assertFalse(patient.getPersonVoided());
 	}
 	
+	@Test
+	public void unvoidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {
+		
+		PatientService patientService = Context.getPatientService();
```

Comment:
there is already a `patientService` available for you. no need to get it from the context